web/frontend/src/components/slam.vue

```diff
@@ -295,101 +354,100 @@ const refresh3dMap = () => {
+              @click="executeMove()"
             />
-            <span class="pr-2">View SLAM Map (3D))</span>
           </div>
-          <div class="float-right pb-4">
-            <div class="flex">
-              <div
-                v-if="show3d"
-                class="w-64"
-              >
-                <p class="font-label mb-1 text-gray-800">
-                  Refresh frequency
-                </p>
-                <div class="relative">
-                  <select
-                    v-model="selected3dValue"
-                    class="
-                      m-0 w-full appearance-none border border-solid border-black bg-white bg-clip-padding px-3 py-1.5
-                      text-xs font-normal text-gray-700 focus:outline-none
-                    "
-                    aria-label="Default select example"
-                    @change="selectSLAMPCDRefreshFrequency()"
-                  >
-                    <option value="manual">
-                      Manual Refresh
-                    </option>
-                    <option value="30">
-                      Every 30 seconds
-                    </option>
-                    <option value="10">
-                      Every 10 seconds
-                    </option>
-                    <option value="5">
-                      Every 5 seconds
-                    </option>
-                    <option value="1">
-                      Every second
-                    </option>
-                  </select>
-                  <div
-                    class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2"
-                  >
-                    <svg
-                      class="h-4 w-4 stroke-2 text-gray-700"
-                      viewBox="0 0 24 24"
-                      stroke="currentColor"
-                      stroke-linejoin="round"
-                      stroke-linecap="round"
-                      fill="none"
-                    >
-                      <path d="M18 16L12 22L6 16" />
-                    </svg>
-                  </div>
+        </div>
+        <div class="justify-start gap-4 sm:border-l border-black">
+        <div v-if="loaded2d && show2d">
+          <div class="flex flex-row pl-5 pt-3">
+            <div class="flex flex-col">
+              <p class="text-xs"> Current Position </p>
+                <div class="flex flex-row">
+                  <p class="pr-2 text-xs items-end"> x </p>
+                  <p >{{ x }}</p>
+                  <p class="pl-7 pr-2 text-xs"> y </p>
+                  <p>{{ y }}</p>
+                  <p class="pl-7 pr-2 text-xs"> z </p>
+                  <p>{{ z }}</p>
                 </div>
-              </div>
-              <div class="px-2 pt-7">
-                <v-button
-                  v-if="show3d"
-                  icon="refresh"
-                  label="Refresh"
-                  @click="refresh3dMap()"
-                />
+            </div>
```

Comment:
these values are not aligned and it looks ugly.
values are aligned when `text-sm`